vercel/resource_project.go

```diff
@@ -579,6 +579,15 @@ At this time you cannot use a Vercel Project resource with in-line ` + "`environ
 				Computed:      true,
 				PlanModifiers: []planmodifier.Bool{boolplanmodifier.UseStateForUnknown()},
 			},
+			"build_machine_type": schema.StringAttribute{
+				Description: "When `on_demand_concurrent_builds` is enabled, choose the build machine: `standard` (4 vCPU, 8 GB) or `enhanced` (8 vCPU, 16GB)",
+				Optional:    true,
+				Computed:    true,
+				Validators: []validator.String{
```

Comment:
Let's figure out how to add a validator to make sure `on_demand_concurrent_builds` is enabled.

Reply:
We have `ModifyPlan` on this resource - we can do it there.